config/settings.yml

```diff
@@ -3,3 +3,4 @@ max_name_length: 50
 max_email_length: 255
 gravatar_size: 80
 per_page: 10
+session_ok: 1
```

Comment:
Is it working, I think param of the checkbox is string type

Reply:
@thanhhungchu95 It works fine.